### ISSUE TITLE
I can't seem to make dunst stop cutting off text.

### ISSUE THREAD

**Issue by that1communist** [dunst-project/dunst#395]
I think this is because of my HiDPi, but i'm not actually sure, i'm on arch, if i make the minimum width absurdly high it works, but I can't make scaling work.

`[global]
    font = "Noto Sans 10"
    shrink = true
    markup = full
    frame_color = "#383838"
    frame_width = 0
    format = "<b>%s</b>\n%b"
    sort = true
    indicate_hidden = true
    alignment = left
    bounce_freq = 0
    show_age_threshold = 60
    word_wrap = true
    ignore_newline = no
    geometry = "0x5-6+30"
    notification_height = 60
    transparency = 0
    idle_threshold = 120
    monitor = 0
    follow = mouse
    sticky_history = true
    browser = /usr/bin/chromium
    line_height = 0
    separator_height = 2
    padding = 8
    horizontal_padding = 20
    separator_color = "#585858"
    show_indicators = false
    startup_notification = false
    icon_position = left
    icon_path = /usr/share/icons/'Arc Red'/actions/16
    max_icon_size = 60

[shortcuts]
    close = ctrl+space
    close_all = ctrl+shift+space
    history = ctrl+grave
    context = ctrl+shift+period

[urgency_low]
    background = "#000000"
    foreground = "#E3C7AF"
    timeout = 5

[urgency_normal]
    background = "#000000"
    foreground = "#E3C7AF"
    timeout = 10

[urgency_critical]
    background = "#000000"
    foreground = "#E3C7AF"
    timeout = 20`

There's my config, just incase this is all on my end.

**Comment by bebehei** [comment]
Could you please give anything to reproduce or a more thorough description of the bug?

**Comment by that1communist** [comment]
I actually can't make it happen with notify-send, it might only happen when I receive a notification with an icon.

I don't have much in the way of reproducibility, but if my theory is correct, my dpi is the issue. Xft.dpi: 140 is in my Xresources, I will do thorough testing when I am free again later today.

**Comment by that1communist** [comment]
Forgot to mention that it always happens with discord. I will look into it's notification system.

**Comment by tsipinakis** [comment]
> I don't have much in the way of reproducibility, but if my theory is correct, my dpi is the issue. Xft.dpi: 140 is in my Xresources, I will do thorough testing when I am free again later today.

The following pieces of information along with the config you provided should be enough to reproduce:
* Output of `dunst --version`
* Output of `dbus-monitor` while one of the broken notifications go through

**Comment by that1communist** [comment]
I fixed it.

the issue was 'font = "Noto Sans 10"' in my config.

I don't know why that breaks it.
`Dunst - A customizable and lightweight notification-daemon v1.2.0-88-gbc9ddf`

**Comment by bebehei** [comment]
Could you please elaborate your problem a bit clearer? We may be able to learn something and mitigate further issues.

Or alternatively you can close the issue if it's fixed.

**Comment by that1communist** [comment]
Well, first I had geometry width 0, shrink enabled. if I enabled wordwrap
after the summary it would go to a newline and the notification wouldn't
expand, if I disabled wordwrap there would be an ellipses even though it
was unnecessary, i tried it again without the quotation marks around my
font choice. Commenting out the font fixes it. I can provide screenshots if
you can't replicate it.

**Comment by bebehei** [comment]
Yes, please provide some screenshots (with the belonging configs). I can't reproduce it.

**Comment by that1communist** [comment]
[global]
    font = "Noto Sans 10"
    markup = full
    frame_color = "#383838"
    frame_width = 0
    format = "<b>%s</b>\n%b"
    sort = true
    indicate_hidden = true
    alignment = right
    bounce_freq = 0
    show_age_threshold = 60
    word_wrap = true
    ellipsize = end
    ignore_newline = false
    geometry = "0x0-200+30"
    transparency = 0
    idle_threshold = 120
    monitor = 0
    follow = mouse
    sticky_history = false
    browser = /usr/bin/chromium
    line_height = 0
    separator_height = 2
    padding = 8
    horizontal_padding = 20
    separator_color = "#585858"
    show_indicators = false
    startup_notification = false
    # https://github.com/knopwob/dunst/issues/26#issuecomment-36159395
    icon_position = left
    icon_path = /usr/share/icons/'Arc Red'/actions/16
    max_icon_size = 60

[shortcuts]
    close = ctrl+space
    close_all = ctrl+shift+space
    history = ctrl+grave
    context = ctrl+shift+period

[urgency_low]
    background = "#000000"
    foreground = "#E3C7AF"
    timeout = 5

[urgency_normal]
    background = "#000000"
    foreground = "#E3C7AF"
    timeout = 10

[urgency_critical]
    background = "#000000"
    foreground = "#E3C7AF"
    timeout = 20
![2017-10-23-191951_2560x1700_scrot](https://user-images.githubusercontent.com/13646790/31917766-4d045662-b827-11e7-9c48-3c6acd9f881c.png)

**Comment by ar013** [comment]
I've had the same issue lately, playing around with different font sizes fixed it for me.

**Comment by eater** [comment]
All else being equal, I am seeing cut-off text with certain font sizes. Arial 11 and Arial 14 are fine; Arial 12 and Arial 13 are cut off. 

I have 
```
geometry = "0x2-40+120"
shrink = yes

```

![2020-05-27 11-12-11 flameshot](https://user-images.githubusercontent.com/106199/83038601-28558880-a00b-11ea-93f6-5f99d3bc6814.png)
![2020-05-27 11-11-50 flameshot](https://user-images.githubusercontent.com/106199/83038604-28558880-a00b-11ea-8f76-9b6633f22d53.png)
![2020-05-27 11-11-30 flameshot](https://user-images.githubusercontent.com/106199/83038605-28ee1f00-a00b-11ea-8ef3-3d95fd436839.png)
![2020-05-27 11-11-07 flameshot](https://user-images.githubusercontent.com/106199/83038606-28ee1f00-a00b-11ea-8e92-b7781ac18818.png)

**Comment by papuSpartan** [comment]
Same issue here

**Comment by fwSmit** [comment]
@papuSpartan @eater There's a potential fix for this in #518. The `geometry` and `shrink` settings are deprecated by it, so you'll have to replace that by a `width = (0,1000)`.

**Comment by papuSpartan** [comment]
> @papuSpartan @eater There's a potential fix for this in #518. The `geometry` and `shrink` settings are deprecated by it, so you'll have to replace that by a `width = (0,1000)`.

Great, i'll try that later.

**Comment by fwSmit** [comment]
@papuSpartan have you had a chance to try #855?
